docker-compose.yml

```diff
@@ -1,6 +1,22 @@
 version: '3.8'
 
 services:
+  frontend-dev:
```

Comment:
Maybe this can be solved in another way, since the two frontend services have almost the same configuration.

It doesn't make sense to have two frontend services in a single compose file (for example, `docker-compose up`) will fail because they both try to bind to port 3001.